test/remote/gateways/remote_midtrans_test.rb

```diff
@@ -12,6 +12,30 @@ def setup
       payment_type: 'credit_card',
       order_id: SecureRandom.uuid
     }
+    @gopay_payment_options = {
+      payment_type: 'gopay',
+      order_id: SecureRandom.uuid,
+      notification_url: 'dummyurl.com',

```

Comment:
Any reason the notification_url is dummyurl.com?

Reply:
No specific reason at all. Just tried passing a dummy URL. Seems like they don't have any validation on the URL passed.

Reply:
If it is a param that is needed to make a payment, should we not accept it as a param rather than giving a dummy URL in all cases? maybe a logic like if passed then we use the variable else an integration URL?

Reply:
It is accepted as a param sent from the integration service and passed along.